src/openadr_ven/constants.py

```diff
@@ -34,5 +34,6 @@
 VEN_ID = "ven_id"
 DISABLE_SIGNATURE = "disable_signature"
 REQUIRED_KEYS = [VEN_NAME, VTN_URL]
+DEMO_VEN = "demo_ven"
```

Comment:
Why is this a string when the agent code expects boolean?

Reply:
`DEMO_VEN` represents the key to be used when parsing the config dictionary. The agent code will get a boolean when the config returns the value of that key:

```python
def _configure_ven_client(.....): ...
    self.add_reports(demo_report=config.get(DEMO_VEN))

def add_reports(self, demo_report=False): 
    .....
```

Reply:
Ok I got it now